adot-testbed/app/src/test/resources/configurations/config-json.yaml

```diff
@@ -0,0 +1,21 @@
+receivers:
+  filelog:
+    include: [/logs/testingJSON.log]
+    encoding: utf-8
+    start_at: beginning
+    operators:
+      - type: json_parser
+
+exporters:
+  logging:
+    verbosity: detailed
+  awscloudwatchlogs:
+    log_group_name: "/test/logs"
+    log_stream_name: "logstream-json"
```

Comment:
can you update all the configurations to set the retention period to a small value like 7 days?

Reply:
We should also use a more descriptive log group and log stream name to make troubleshooting easier.

Reply:
Consider adding a unique identifier for the test run for each log stream.

Reply:
Agreed, added unique identifier